packages/docs/docs/behaviour/obliqueVoxels.md

```
@@ -0,0 +1,249 @@
+2. **Integer oblique ownership / iteration** — used for fast segmentation voxel
+   iteration. Uses integer `u, v, w`. This page is about (2).
+
+## The unimodular integer oblique basis
+
+### Definition
+
+Given a volume with an (orthonormal) direction matrix and voxel spacing, and a
+world-space view plane normal, the iterator builds an **integer** basis
+`[A B C]` and a primitive integer plane normal `N = [ni, nj, nk]` such that:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)